src/test/unit_tests/jest_tests/test_nc_nsfs_bucket_cli.test.js

```diff
@@ -162,6 +205,116 @@ describe('manage nsfs cli bucket flow', () => {
+
+        const account_defaults2 = {
+            name: account_name2,
+            new_buckets_path: `${root_path}new_buckets_path_user41/`,
+            uid: 1002,
+            gid: 1002,
+            access_key: 'GIGiFAnjaaE7OKD5N7hY',
+            secret_key: 'G2AYaMpU3zRDcRFWmvzgQr9MoHIAsD+3pEXAMPLE',
+        };
+
+        const bucket_defaults = {
+            name: bucket_name,
+            owner: account_name,
+            path: bucket_storage_path,
+        };
+
+
+        beforeEach(async () => {
+            await fs_utils.create_fresh_path(`${config_root}/${CONFIG_SUBDIRS.BUCKETS}`);
+            await fs_utils.create_fresh_path(root_path);
+            await fs_utils.create_fresh_path(bucket_storage_path);
+            let action = ACTIONS.ADD;
+            // account add 1
+            let { new_buckets_path: account_path } = account_defaults;
+            let account_options = { config_root, ...account_defaults };
+            await fs_utils.create_fresh_path(account_path);
+            await fs_utils.file_must_exist(account_path);
+            await set_path_permissions_and_owner(account_path, account_options, 0o700);
+            await exec_manage_cli(TYPES.ACCOUNT, action, account_options);
+            // account add 2
+            account_path = account_defaults2.new_buckets_path;
+            account_options = { config_root, ...account_defaults2 };
+            await fs_utils.create_fresh_path(account_path);
+            await fs_utils.file_must_exist(account_path);
+            await set_path_permissions_and_owner(account_path, account_options, 0o700);
+            await exec_manage_cli(TYPES.ACCOUNT, action, account_options);
+            // bucket add
+            action = ACTIONS.ADD;
+            const bucket_options = { config_root, ...bucket_defaults};
+            await fs_utils.create_fresh_path(bucket_options.path);
+            await fs_utils.file_must_exist(bucket_options.path);
+            await set_path_permissions_and_owner(bucket_options.path, account_defaults, 0o700);
+            await exec_manage_cli(TYPES.BUCKET, action, bucket_options);
+        });
+
+        afterEach(async () => {
+            await fs_utils.folder_delete(`${config_root}`);
+            await fs_utils.folder_delete(`${root_path}`);
+        });
+
+        it('should fail - cli update bucket owner - owner does not have any permission to path', async () => {
+            const action = ACTIONS.UPDATE;
+            const bucket_options = { config_root, name: bucket_defaults.name, owner: account_defaults2.name};
+            await fs_utils.create_fresh_path(bucket_defaults.path);
+            await fs_utils.file_must_exist(bucket_defaults.path);
+            await set_path_permissions_and_owner(bucket_defaults.path, account_defaults2, 0o077);
+            const res = await exec_manage_cli(TYPES.BUCKET, action, bucket_options);
+            expect(JSON.parse(res.stdout).error.message).toBe(ManageCLIError.InaccessibleBucketPath.message);
```

Comment:
Checking error code is more common practice and safe.

Reply:
I will change, thanks.
Would you please explain about this more (or send a link) - why is it more safe?

Reply:
chances of updating error message is more than error code, and adding a space or dot in error message might break complete test cases. And here error code is each error have distinct error code.

Reply:
Thank you, I just don't fully understand - it is always checks for the same line, right?
For example in this line:
`expect(JSON.parse(res.stdout).error.message).toBe(ManageCLIError.InaccessibleBucketPath.message);` (before the change) it would compare this line `message: 'Account should have read & write access to the specified bucket storage path',` so by changing this line, the test should pass.